### PR TITLE
Process Diff Blessed: add diff role and helps

### PR DESCRIPTION
**Pull request by ebelter**

### CODE REVIEW

File: lib/perl/Genome/Role/Comparable/Command/DiffBlessed.pm [hunk 1]
```diff
@@ -11,6 +11,14 @@ role Genome::Role::Comparable::Command::DiffBlessed {
     requires => ['new_object'],
 };
 
+sub help_brief : Overrides(Genome::Role::Comparable::Command::Diff) {
```
Comment (tmooney) [inline lib/perl/Genome/Role/Comparable/Command/DiffBlessed.pm]: I'm not sure what it means for this to specify an override for a role that it does not itself consume.

**Comment by tmooney** [comment]
I think I'm of the opinion that inheritance was the better relationship for the build and process diff commands.

File: lib/perl/Genome/Role/Comparable/Command/DiffBlessed.pm (hunk 1 above)
Comment (ebelter) [inline lib/perl/Genome/Role/Comparable/Command/DiffBlessed.pm]: This is an artifact of me trying to let this role have the 'diff' role. The methods from the diff role were not conferred to the diff-blessed role, and I abandoned this fix. I will remove it, and have additionally fixed the help to not refer to processes, but objects.

**Comment by ebelter** [comment]
@tmooney I agree. I wanted to see what this looked like mocked up. Close this PR, and revert b251a2a9eed11c399251381776550e2aa6b6b2fa ?

**Comment by ebelter** [comment]
This change has horked some additional tests...

**Comment by ebelter** [comment]
Closing in favor of #1181.

**Pull request closed without merging**